audio/Makefile

```diff
@@ -38,10 +38,10 @@ LINUXHEADERSDIR := ../third_party/fx2lib-linux-headers
 COMMON_DIR := ../common
 INCS := -I$(LINUXHEADERSDIR)
 
-CFLAGS := -DDSCR_AREA=0x3e00 --code-size 0x3c00 --xram-size 0x0200
+CFLAGS := -DDSCR_AREA=0x3d00 --code-size 0x3c00 --xram-size 0x0200
```

Comment:
Why 0x3d00? I assume you need more space for the descriptors?

Maybe add a comment here?

Reply:
Yup. This was the result of our debug session a while ago

Reply:
Fixed